data/crac-io/crac-io-json/src/test/resources/retrocompatibility/v2/crac-v2.5.json

```diff
@@ -0,0 +1,513 @@
+      "instant" : "curative",
+      "contingencyId" : "contingency1Id",
+      "usageMethod" : "available"
+    } ],
+    "networkElementIdsAndKeys" : {
+      "generator1Id" : 1.0,
+      "generator2Id" : -1.0
+    },
+    "initialSetpoint" : 50,
+    "ranges" : [ {
+      "min" : -500.0,
+      "max" : 500.0
+    }, {
+      "min" : -1000.0,
+      "max" : 1000.0
+    } ]
+  } ],
+  "counterTradeRangeActions" : [ {
+    "id" : "counterTradeRange1Id",
+    "name" : "counterTradeRange1Name",
+    "operator" : null,
+    "speed" : 30,
+    "onInstantUsageRules" : [ {
+      "instant" : "preventive",
+      "usageMethod" : "available"
+    } ],
+    "onConstraintUsageRules" : [ {
+      "instant" : "preventive",
+      "cnecId" : "cnec3curId",
+      "usageMethod" : "available"
+    }, {
+      "instant" : "curative",
+      "cnecId" : "voltageCnecId",
+      "usageMethod" : "available"
+    }, {
+      "instant" : "curative",
+      "cnecId" : "angleCnecId",
+      "usageMethod" : "available"
+    } ],
+    "onContingencyStateUsageRules" : [ {
+      "instant" : "curative",
+      "contingencyId" : "contingency1Id",
+      "usageMethod" : "available"
+    } ],
+    "exportingCountry" : "FR",
+    "importingCountry" : "DE",
+    "initialSetpoint" : 50,
+    "ranges" : [ {
+      "min" : -500.0,
+      "max" : 500.0
+    }, {
+      "min" : -1000.0,
+      "max" : 1000.0
+    } ]
+  } ],
+  "networkActions" : [ {
+    "id" : "complexNetworkActionId",
+    "name" : "complexNetworkActionName",
+    "operator" : "RTE",
+    "speed" : 40,
+    "onInstantUsageRules" : [ {
+      "instant" : "preventive",
+      "usageMethod" : "available"
+    } ],
+    "terminalsConnectionActions" : [ {
+      "networkElementId" : "ne1Id",
+      "actionType" : "close"
+    } ],
+    "phaseTapChangerTapPositionActions" : [ {
+      "networkElementId" : "pst",
+      "tapPosition" : 5
+    } ]
+  }, {
+    "id" : "injectionSetpointRaId",
+    "name" : "injectionSetpointRaName",
+    "operator" : "RTE",
+    "onConstraintUsageRules" : [ {
+      "instant" : "auto",
+      "cnecId" : "cnec3autoId",
+      "usageMethod" : "forced"
+    } ],
+    "generatorActions" : [ {
+      "networkElementId" : "injection",
+      "activePowerValue" : 260.0
+    }]
+  }, {
+    "id" : "injectionSetpointRa2Id",
+    "name" : "injectionSetpointRa2Name",
+    "operator" : "RTE",
+    "onFlowConstraintInCountryUsageRules" : [ {
+      "instant" : "curative",
+      "contingencyId" : "contingency2Id",
+      "country" : "FR",
+      "usageMethod" : "available"
+    } ],
+    "loadActions" : [ {
+      "networkElementId" : "LD1",
+      "activePowerValue" : 260.0
+    }],
+    "danglingLineActions" : [ {
+      "networkElementId" : "DL1",
+      "activePowerValue" : -120.0
+    }],
+    "switchActions" : [ {
+      "networkElementId" : "BR1",
+      "actionType" : "open"
+    } ]
+  }, {
+    "id" : "injectionSetpointRa3Id",
+    "name" : "injectionSetpointRa3Name",
```

Comment:
It might be worth it to rename all the actions to better reflect what they are

Reply:
Its the same name than 2.4 to show the transformation between 2.4 and 2.5, so the name was already not correct from 2.4 from previous PR https://github.com/powsybl/powsybl-open-rao/pull/920. So I changed it in previous PR and to be clearer I created a new network action for my tests instead of modifying the old ones. Name should be appropriate now.